### DISCUSSION TITLE
Windows 10: Server started and nothing happens.

### DISCUSSION THREAD

**Discussion by chiphu** [devld/go-drive#52] [General]
I don't know why? Please help me out! Thanks.

![image](https://user-images.githubusercontent.com/90211256/210311992-3ec8f68c-c9e0-499f-8e59-e4085bcbe634.png)

**Comment by devld** [comment]
It started. You can visit [http://localhost:8089](http://localhost:8089).

The default user is `admin`, its password is `123456`

**Comment by chiphu** [comment]
> It started. You can visit http://localhost:8089.
> 
> The default user is `admin`, its password is `123456`

Thank you very much! Now I gotta see and translate the doc for the other functions.
